src/components/features/accordion/alis/Object.vue

```diff
@@ -0,0 +1,199 @@
+import { useFetch } from "@vueuse/core";
+
+import { upperFirst } from "lodash";
+
+const props = defineProps({
+  feature: {
+    type: Object,
+    default: () => ({}),
+  },
+  getValue: {
+    type: Function,
+    default: () => {},
+  },
+  getCategory: {
+    type: Function,
+    default: () => {},
+  },
+  getCadastralId: {
+    type: Function,
+    default: () => {},
+  },
+});
+
+const feature = computed(() => props.feature);
+const cadastralId = computed(() => props.getCadastralId(feature.value));
+const hasCadastralId = computed(() => !!cadastralId.value);
+
+function isLoading(activeTab: string) {
+  if (activeTab === "info") {
+    return !!alisWaterBodyLoading.value;
+  } else if (activeTab === "fishstocking") {
+    return !!zuvinimasInfoLoading.value;
+  }
+
+  return false;
+}
+
+const { data: alisWaterBody, isFetching: alisWaterBodyLoading } = useFetch<any>(
+  getWaterBodyInfoUrl(cadastralId.value),
+  { immediate: hasCadastralId.value }
+).json();
+
+const { data: zuvinimasInfo, isFetching: zuvinimasInfoLoading } = useFetch<any>(
+  getFishTypesInfoByYearUrl({ cadastralId: cadastralId.value }),
+  {
+    immediate: hasCadastralId.value,
+    afterFetch(ctx: any) {
+      ctx.data = ctx.data[cadastralId.value] || {};
+      return ctx;
+    },
+  }
+).json();
+
+const tabs = [
+  { type: "info", name: "Bendra info" },
+  { type: "fishstocking", name: "Žuvų įveisimas" },
+];
+
+function getFeatureProp(name: string | string[]) {
+  return props.getValue(feature.value, name);
+}
+const basicInfo = [
+  { key: "Pavadinimas", value: () => getFeatureProp("Pavadinimas") },
+  {
+    key: "UETK kadastro ID",
+    value: () => cadastralId.value,
+  },
+  { key: "Kategorija", value: () => props.getCategory(feature.value) },
+  { key: "Savivaldybė", value: () => alisWaterBody.value?.savivaldybe?.pavadinimas },
+  {
+    key: "Centro X koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Centro X koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Centro Y koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Centro Y koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Žiočių X koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Žiočių X koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Žiočių Y koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Žiočių Y koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Vandens paviršiaus be salų plotas, ha",
+    value: () => getFeatureProp("Vandens paviršiaus be salų plotas, ha"),
+  },
+  {
+    key: "Kranto linijos ilgis, km",
+    value: () => getFeatureProp("Kranto linijos ilgis, km"),
+  },
+  {
+    key: "Statusas",
+    value: () =>
+      hasCadastralId.value
+        ? `${alisWaterBody.value?.isnuomotas ? "Išnuomotas" : "Neišnuomotas"} telkinys`
+        : "",
+  },
+  {
+    key: "Limituota žvejyba?",
+    value: () =>
+      hasCadastralId.value ? (alisWaterBody.value?.licencija ? "Taip" : "Ne") : "",
+  },
+  {
+    key: "Google žemėlapis",
+    link: "Nuoroda į Google",
```

Comment:
Missing underline to look like a href link. Also, better name would be `Nuoroda į Google Žemėlapį`